server/routes/students.js

```diff
@@ -0,0 +1,62 @@
+const router = require('express').Router();
+const { Student, Test } = require('../db/models');
+
+// GET /api/students
+router.get('/', async (req, res, next) => {
+  try {
+    const students = await Student.findAll({
+      include: { all: true }
```

Comment:
ik it was like this in the original, but I think it might be more clear for SS to use `include: Test` instead of `all: true` since a) there's only 1 model that could be included, so "all" is a little overkill, and b) we don't cover the "all" option

Reply:
Updated to `include: Test`.